command/operator_debug.go

```diff
@@ -876,6 +897,34 @@ func (c *OperatorDebugCommand) collectAgentHost(path, id string, client *api.Cli
 	c.writeJSON(path, "agent-host.json", host, err)
 }
 
+func (c *OperatorDebugCommand) collectPeriodicPprofs(client *api.Client) {
```

Comment:
Hi @danishprakash! I had a chat with @davemay99 about this and he pointed out that when we're collecting the periodic pprofs like this, we don't differentiate the files that we're writing on disk for each interval. So the pprof for each interval will just end up overwriting the one before it!

We'll need to thread an ID or interval index or something down to the `collectPrrofs` function in order to avoid that. Once that's done, I think this should be ready to go.

Reply:
Hey @tgross, I've propagated the interval count to be used as a suffix for the filename. Could've used an ID too but felt like additional overhead to generate and make sense of a UID of some sort. Let me know what you think? Thanks